assets/js/src/settings.js

```diff
@@ -122,7 +122,11 @@
 			}
 		}
 
-		jQuery.post( ajaxurl, { action: 'rsa_ip_check', 'ip_address': ip, 'ip_address_comment': comment }, function(response) {
+		jQuery.post( ajaxurl, {
+			action: 'rsa_ip_check',
+			'ip_address': ip,
+			nonce: rsaAdmin.nonce
```

Comment:
Just wanted to leave a note here that in adding the nonce the `ip_address_comment` was lost, probably a merge casualty.